mautic/mautic
# Asset download segment filter

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? |  Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

#### Description:

Adds asset multi selector support. 
Adds segment filter for leads having downloaded an asset.

#### Steps to test this PR:

1. Create an asset.
2. Download the asset as a lead.
3. Create segment selecting the created asset.
3. Update the segment and check if it contains the lead.
